Core/Util/GameManager.h

```diff
@@ -64,11 +66,15 @@ class GameManager {
 	}
 
 	// Only returns false if there's already an installation in progress.
-	bool InstallGameOnThread(std::string zipFile, bool deleteAfter);
+	bool InstallGameOnThread(std::string url, std::string tempFileName, bool deleteAfter);
 
 private:
-	bool InstallGame(std::string zipfile, bool deleteAfter = false);
+	bool InstallGame(std::string url, std::string tempFileName, bool deleteAfter);
```

Comment:
nit: Any reason not const & for this and InstallRawISO?

-[Unknown]

Reply:
Just laziness, fixing.